crates/hc_bundle/README.md

```diff
@@ -6,42 +6,48 @@
 
 [![License: Apache-2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
-Utilities to create DNA and hApp bundle files from a source working directory, and vice-versa.
-This crate defines two separate subcommands for the `hc` CLI tool, one for each type of bundle.
-Both subcommands are very similar and have identical interfaces.
+Utilities to create DNA, hApp, and web-hApp bundle files from a source working directory and manifest file, and vice-versa.
+This crate defines three separate subcommands for the `hc` CLI tool, one for each type of bundle.
+All subcommands are very similar and have nearly identical flags and options.
 
-This crate also defines standalone binaries for each subcommand, `hc-dna` and `hc-app`.
+This crate also defines standalone binaries for each subcommand, `hc-dna`, `hc-app`, and `hc-web-app`.
 
 Usage instructions from the `-h` flag:
 
 ```sh
 $ hc dna -h
-
-hc-dna 0.0.1
-Work with DNA bundles
+holochain_cli_bundle 0.1.3
+Work with Holochain DNA bundles
 
 USAGE:
-    hc dna <SUBCOMMAND>
+    hc-dna <SUBCOMMAND>
 
 FLAGS:
     -h, --help       Prints help information
     -V, --version    Prints version information
 
 SUBCOMMANDS:
     help      Prints this message or the help of the given subcommand(s)
-    init      Create a new, empty Holochain DNA bundle working directory
-    pack      Pack the contents of a directory into a `.dna` bundle file
-    unpack    Unpack the parts of `.dna` file out into a directory
+    init      Create a new, empty Holochain DNA bundle working directory and create a new sample `dna.yaml` manifest
+              inside. 
+    pack      Pack into the `[name].dna` bundle according to the `dna.yaml` manifest, found inside the working
+              directory. The `[name]` is taken from the `name` property of the manifest file
+    unpack    Unpack parts of the `.dna` bundle file into a specific directory
 ```
 
-`hc app -h` is very similar.
+`hc app` and `hc web-app` are very similar, only differing by the addition of a `--recursive` flag.
+If used, this flag attempt to first pack all the assets to be included in the bundle being packed.
+If it doesn't find the bundled DNA or hApp asset specified, it will by convention look for a
```

Comment:
:+1: correct

Reply:
phew, I just wasn't sure. Thanks for the confirmation.